facesign-service/Dockerfile

```
@@ -132,8 +132,12 @@ RUN sed -i 's#search2D2DFacePortraitDatabaseRootDirectory: .*#search2D2DFacePort
 RUN sed -i 's#usageLogServerUri: .*#usageLogServerUri: http://localhost:3000/#' ./config.yaml
 RUN printf "#!/bin/bash\njava -cp $HOME_FACETEC_SERVER_CORE/libs/FaceTecSDK-Server-Core-$FACETEC_SDK_VERSION.jar:FaceTec-Server-Webservice.jar com.facetec.facetecserver.app.App" > ./run.ash
 
-# Add feature flags for max 5 attempts
 RUN echo "" >> ./config.yaml
+
+# Add max simultaneous requests
+RUN echo "maxSimultaneousRequests: 50" >> ./config.yaml
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)